### PR TITLE
Loginclient

### PR DESCRIPTION
**Pull request by halakhamis07**

### CODE REVIEW

File: src/models/clients.models.ts [hunk 1]
```diff
@@ -7,24 +7,29 @@ export function fetchClients() {
 }
 
 export function getClient(data) {
-  const gov_id = [data.gov_id];
+<<<<<<< Updated upstream
```
Comment (Karyum) [inline src/models/clients.models.ts]: There is a merge conflict here, you need to fix it

File: src/models/clients.models.ts [hunk 2]
```diff
@@ -7,24 +7,29 @@ export function fetchClients() {
...
-    VALUES ($1,$2,$3)`,
-    treatment
-  );
+  // const govId = await addClient(data.client);
+  // const treatment = [govId, client.protocolId, client.startDate];
+  // return db.query(
+  //   `INSERT INTO treatment (client_id,protocol_id,start_date)
+  //   VALUES ($1,$2,$3)`,
+  //   treatment
+  // );
```
Comment (Karyum) [inline src/models/clients.models.ts]: If you are not using this code then remove it

File: src/controllers/clients.controller.ts [hunk 3]
```diff
@@ -19,13 +19,14 @@ const allClients = catchAsync(async (req: any, res: any) => {
...
   const { gov_id, passcode } = req.body;
-
+  console.log(req.body);
   if (!gov_id || !passcode) {
     throw new ApiError(httpStatus.BAD_REQUEST, "Missing data");
   }
 
   const client = await getClient(gov_id);
-  if (client.length) {
+  console.log(client);
```
Comment (Karyum) [inline src/controllers/clients.controller.ts]: remove the console log

**Review by Karyum — CHANGES_REQUESTED** [review]

File: src/models/clients.models.ts (hunk 2 above)
Comment (halakhamis07) [reply src/models/clients.models.ts]: this is george's

**Review by Karyum — APPROVED** [review]

**Pull request merged**